src/main/resources/application.yaml

```diff
@@ -87,7 +87,7 @@ idam:
     url: ${S2S_URL:http://rpe-service-auth-provider-aat.service.core-compute-aat.internal}
     pbaFromUserAccountMap: false
   s2s-authorised:
-    services: ${PRD_S2S_AUTHORISED_SERVICES:rd_professional_api,rd_user_profile_api,xui_webapp,finrem_payment_service,fpl_case_service,iac}
+    services: ${PRD_S2S_AUTHORISED_SERVICES:rd_professional_api,rd_user_profile_api,xui_webapp,finrem_payment_service,fpl_case_service,iac,aac-manage-case-assignment}
   api.url: ${idam_url:https://idam-api.aat.platform.hmcts.net}
```

Comment:
there should be test case for the new service added
There should also be a negative test case to prove that another service which is not part of above list cannot access prd apis

Reply:
This has been addressed in the following commit: https://github.com/hmcts/rd-professional-api/pull/557/commits/b96afdd682c4793a39e34658477ff67ee473b289